nautobot/ipam/tests/test_models.py

```diff
@@ -581,13 +577,10 @@ def test_get_utilization(self):
         )
 
         # 25% utilization
-        prefixes = [
-            Prefix(prefix="22.0.0.0/12", status=self.status, namespace=self.namespace),
-            Prefix(prefix="22.16.0.0/12", status=self.status, namespace=self.namespace),
-            Prefix(prefix="22.32.0.0/12", status=self.status, namespace=self.namespace),
-            Prefix(prefix="22.48.0.0/12", status=self.status, namespace=self.namespace),
-        ]
-        [p.save() for p in prefixes]  # pylint: disable=expression-not-assigned
+        Prefix.objects.create(prefix="22.0.0.0/12", status=self.status, namespace=self.namespace)
+        Prefix.objects.create(prefix="22.16.0.0/12", status=self.status, namespace=self.namespace)
+        Prefix.objects.create(prefix="22.32.0.0/12", status=self.status, namespace=self.namespace)
+        Prefix.objects.create(prefix="22.48.0.0/12", status=self.status, namespace=self.namespace)
```

Comment:
IIRC the reason we were doing `save()` instead of `create()` was because the logic to decompose `prefix` into its components wasn't happening in some cases - is that a non-issue here?

Reply:
It looks like it's doing almost the same thing by using create:

https://github.com/django/django/blob/722e9f8a38f5b34f2423059a75f8a59bb8eb931a/django/db/models/query.py#L446-L454
```py
    def create(self, **kwargs):
        """
        Create a new object with the given kwargs, saving it to the database
        and returning the created object.
        """
        obj = self.model(**kwargs)
        self._for_write = True
        obj.save(force_insert=True, using=self.db)
        return obj
```